### ISSUE TITLE
Identity columns (Postgres 10+) not supported

### ISSUE THREAD

**Issue by diffen** [jakob/Postico#492]
Postgres 10 introduced identity columns that are better than serial. This blog post explains them:
https://blog.2ndquadrant.com/postgresql-10-identity-columns/

These don't seem to be supported by Postico when creating a new table. So there's basically 2 bugs:

1. These columns are not supported when creating a table.
2. These columns are not supported when viewing structure of a table created via SQL that uses identity columns.

## What did you do?

I created a table like so:
```
CREATE TABLE test_new (
    id int GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
    payload text
);
```

In psql, `\d test_new` shows this in the Default column: `generated by default as identity`

## What did you expect to happen?

In Postico, in the Structure view of that table, in the id column, Default description, it should mention that this is an identity column. Similar to what \d test_new shows you in psql. 

## What actually happened?
It shows id correctly with integer data type and primary key. But it gives no indication that this is an identity column.

## What software versions are you using?

Postico version:  
Version 1.3.1 (2153)

macOS version:  
10.13.3

PostgreSQL version:  
10.1

**Comment by diffen** [comment]
Bumping this thread to see if there's an update planned for supporting this data type.

**Comment by bfelbo** [comment]
@jakob Any plans to add support for this? It's now more than 2.5 years since Postgres 10 was released and many people are using identity columns for all their tables.

**Comment by jakob** [comment]
We just released Postico 1.5.20 with support for identity columns!
